src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptorUnitTest.java

```diff
@@ -451,71 +449,97 @@ public void testGetClassForPluginHelpFromReflection(){
         Assert.assertTrue(pluginDescriptor.getClassForPluginHelp("Foo")==null);
     }
 
+    @Test
+    public void testUseAllAnnotations(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {"--"+StandardArgumentDefinitions.ENABLE_ALL_ANNOTATIONS};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==ChromosomeCounts.class));
+    }
+
+    @Test
+    public void testIncludeAllExcludeIndividual(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {"--"+StandardArgumentDefinitions.ENABLE_ALL_ANNOTATIONS, "-AX", "Coverage"};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==ChromosomeCounts.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==AS_RMSMappingQuality.class));
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==AS_StandardAnnotation.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==RMSMappingQuality.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==QualByDepth.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==ReadPosRankSumTest.class));
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==StrandOddsRatio.class));
```

Comment:
Again, this doesn't seem like a sufficient test. Use a `ClassFinder` to find all annotations manually, remove `Coverage`, and assert equality regardless of ordering.